backend/api/Areas/Leases/Mapping/Lease/SecurityDepositMap.cs

```diff
@@ -0,0 +1,25 @@
+using Mapster;
+using Entity = Pims.Dal.Entities;
+using Model = Pims.Api.Areas.Lease.Models.Lease;
+
+namespace Pims.Api.Areas.Lease.Mapping.Lease
+{
+    public class SecurityDepositMap : IRegister
+    {
+        public void Register(TypeAdapterConfig config)
+        {
+            config.NewConfig<Entity.SecurityDeposit, Model.SecurityDepositModel>()
+                .Map(dest => dest.Id, src => src.Id)
+                .Map(dest => dest.RowVersion, src => src.RowVersion)
+                .Map(dest => dest.SecurityDepositHolderTypeId, src => src.SecurityDepositHolderTypeId)
+                .Map(dest => dest.SecurityDepositHolderType, src => src.SecurityDepositHolderType.Description)
+                .Map(dest => dest.SecurityDepositTypeId, src => src.SecurityDepositTypeId)
+                .Map(dest => dest.SecurityDepositType, src => src.SecurityDepositType.Description)
```

Comment:
The types could be mapped directly having the model like:
`public TypeModel<string> SecurityDepositType{ get; set; }`
`.Map(dest => dest.SecurityDepositType, src => src.SecurityDepositType)`

Reply:
See my previous comment. Been following an existing pattern. Happy to discuss as a team what approach is best